battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/model/response/Register.java

```diff
@@ -0,0 +1,15 @@
+package dreamteam.battleship.service.springcontroller.model.response;
+
+import dreamteam.battleship.service.springcontroller.model.Player;
+
+/**
+ * Created by ehsan on 18.07.16.
```

Comment:
When you do Javadocs on class, capitalize your name, and beside who created the class and when, add actual Javadoc.